src/phpDocumentor/Descriptor/GuideSetDescriptor.php

```diff
@@ -13,8 +13,15 @@
 
 namespace phpDocumentor\Descriptor;
 
+use phpDocumentor\Dsn;
+
 final class GuideSetDescriptor extends DocumentationSetDescriptor
 {
+    /**
+     * @param array<Dsn|list<string>> $source
```

Comment:
Oh? I need to check where I made a mistake. I don't think I meant this to be either